functions/pylint/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:alpine3.20
```

Comment:
@ColinMcNeil let's merge this but pylint is one of the tools that is getting swept up into this giant 100 lint nixpkgs -> Docker extravaganza.  So ultimately, what we really need is to gather requriements for wrappers like lint.sh and output_format.sh.

In this mode, I think you have designed three different modes (json, summary, and condensed).  Is that right?

Reply:
>None of these are being written as artifacts yet, but this all kind of makes us wonder whether there are some general patterns here

I really think so. There are four output modes I found:

- json: whatever the linter output in json format
- condensed: A list of filenames grouped by violations. This contains a ton of relevant info for bulk fixes, but in a very small payload
- complaints: A list of complaints with keys `path` `start` `end` and `violation`
- Summary: Great for human readable, just says `x violations found across y files`

Reply:
>lint.sh

At the bare minimum, we need to convert JSON args to CLI args. I would implement

```json
{
  args: "" // Any additional args to pass to the tool
}
```

And lint.sh can also include default arguments to help run the tool. In the instance of both pylint AND eslint, a json format flag is passed in. That isn't something we need to instruct the LLM to do.